build.gradle

```diff
@@ -66,3 +74,7 @@ jar {
 
     version = null
 }
+
+application {
+    mainClassName 'com.exactpro.th2.codec.hand.Controller'
+}
```

Comment:
Is it a random change?

Reply:
Nope, without that gradle build command failed with following error:

A problem was found with the configuration of task ':startScripts' (type 'CreateStartScripts').
> No value has been specified for property 'mainClassName'.

All our components contain this property, act, for example.